moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -26,14 +26,21 @@ const makeAdvancedSearchWhereFilters = (filters) =>
 
       if (!filterConfigForField) {
         throwFallbackComponentError(
-          `Field ${field} in this url no longer exists in the project list.`
+          `Field ${field} in this URL no longer exists in the project list.`
         );
       }
 
       const { type } = filterConfigForField;
 
       // Use operator name to get the GraphQL operator config for that operator
       const operatorConfig = FILTERS_COMMON_OPERATORS[operator];
+
+      if (!operatorConfig) {
+        throwFallbackComponentError(
+          `Operator ${operator} in this URL no longer exists in the project list.`
```

Comment:
Maybe stick a "The" on the front of the sentence here? Couldn't be less of a big deal though.

Reply:
Im gonna go ahead and merge bc i think we may be changing up the wording of this component anyways in some future improvements? :thinking: